spec/yaml/swiftnav/sbp/integrity.yaml

```diff
@@ -82,9 +202,12 @@ definitions:
         - obs_time:
             type: GPSTimeSec
             desc: GNSS reference time of the observation used to generate the flag.
-        - corr_time:
+        - atmo_corr_time:
             type: GPSTimeSec
-            desc: GNSS reference time of the correction associated to the flag.
+            desc: GNSS reference time of the atmospheric correction associated to the flag.
+        - sat_corr_time:
+            type: GPSTimeSec
+            desc: GNSS reference time of the satellite correction associated to the flag.
```

Comment:
Message stores two correction time:
- one for sat corr
- one for atmo (iono + tropo) corr